### PR TITLE
Delete directory after delete all images

### LINKED ISSUE #67: Delete all nested dirrectories.
After delete all images from model, who related to gallery I can see folder naming id of model. I think we need delete this folder too.
Your opinions?

### PR DESCRIPTION
**Pull request by bscheshirwork**
Nested of #65 

Fix #67  

NOTE: This new behavior with remove folder MUST have a line on changelog.

### CODE REVIEW

File: GalleryBehavior.php [hunk 1]
```diff
@@ -223,6 +222,27 @@ public function getFilePath($imageId, $version = 'original')
         return $this->directory . '/' . $this->getFileName($imageId, $version);
     }
 
+    public function getDirectoryPath()
```
Comment (zxbodya) [inline GalleryBehavior.php]: should this be private?

File: GalleryBehavior.php [hunk 2]
```diff
@@ -314,6 +348,9 @@ function ($image) use (&$removed) {
                 }
             );
         }
+        if (is_null($this->_images)) {
+            $this->removeDirectory($this->getDirectoryPath());
+        }
```
Comment (zxbodya) [inline GalleryBehavior.php]: this is not correct - `is_null($this->_images)` means only that `getImages()` method was not called yet

File: GalleryBehavior.php (hunk 1 above)
Comment (bscheshirwork) [reply GalleryBehavior.php]: IMHO This functional can be use outside widget for build the path to galery folder of model

Comment (bscheshirwork) [reply GalleryBehavior.php]: Also we can redefine it in children

**Pull request closed without merging**